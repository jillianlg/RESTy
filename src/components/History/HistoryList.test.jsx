import React from 'react';
import { render, cleanup } from '@testing-library/react';
import HistoryList from './HistoryList';

describe('HistoryItem component', () => {
  afterEach(() => cleanup());
  it('renders a history item', () => {
    const { asFragment } = render(
      <HistoryList
        history={[]}
      />);
    expect(asFragment()).toMatchSnapshot();
  });
});