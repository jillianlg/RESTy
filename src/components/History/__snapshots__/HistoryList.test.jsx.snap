// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryItem component renders a history item 1`] = `
<DocumentFragment>
  <ul
    class="History"
  />
</DocumentFragment>
`;
